client/src/app/site/motions/modules/motion-detail/components/motion-detail/motion-detail.component.ts

```diff
@@ -567,6 +561,29 @@ export class MotionDetailComponent extends BaseComponent implements OnInit, OnDe
         });
     }
 
+    private registerSubjects(): void {
+        // get required information from the repositories
+        this.tagObserver = this.tagRepo.getViewModelListBehaviorSubject();
```

Comment:
If you do not need `this.tagObserver`, ..., I have a proposal: Make an array of repos and loop through them.

Reply:
you need to access the subjects in the view aswell.